docs_user/modules/proc_migrating-mgr-from-controller-nodes.adoc

```diff
@@ -71,25 +63,25 @@ placement:
 ----
 
 . Save the spec in the `/tmp/mgr.yaml` file.
-. Apply the spec with cephadm by using the orchestrator:
+. Apply the spec with `cephadm`` by using the Ceph Orchestrator:
 +
 ----
-sudo cephadm shell -m /tmp/mgr.yaml -- ceph orch apply -i /mnt/mgr.yaml
+$ sudo cephadm shell -m /tmp/mgr.yaml -- ceph orch apply -i /mnt/mgr.yaml
 ----
-+
-As a result of this procedure, you see a Ceph Manager daemon count that matches
-the number of hosts where the `mgr` label is added.
```

Comment:
“The Ceph Manager daemon count should match the number of hosts where the `mgr` label is added.” - Does this mean that the ‘mgr’ label has a number in its place, e.g., label: 10, or is there a line under “label: mgr”?